crt_portal/cts_forms/templates/forms/report_base.html

```diff
@@ -112,5 +112,11 @@ <h2 class="margin-0" tabindex="0">{{ current_step_title }}</h2>
 {% endblock %}
 
 {% block page_js %}
+<script type="text/javascript">
```

Comment:
The report_base is inherited by all pages of the form. we can move the word count to the specific templates that need it but my guess is that it would not be noticeably more performant etc. If I am wrong, we can move it.

Reply:
Got it! I mentioned this because of this error that I see on the protected class page:

<img width="734" alt="Screen Shot 2019-12-03 at 5 08 05 PM" src="https://user-images.githubusercontent.com/1421848/70093953-815dda00-15ef-11ea-8b29-2f3d3bdc2005.png">